contracts/src/interfaces/IRenzo.sol

```diff
@@ -58,7 +58,7 @@ interface IRenzoOracle {
         uint256 _ezETHBeingBurned,
         uint256 _existingEzETHSupply,
         uint256 _currentValueInProtocol
-    ) external pure returns (uint256);
+    ) external view returns (uint256);
```

Comment:
Why'd we change this? It should be pure unless I'm missing something: https://github.com/Renzo-Protocol/contracts-public/blob/0530cbd20590677da5fa6fd32a0a8b250a9eb028/contracts/Oracle/RenzoOracle.sol#L134

Reply:
I updated to use the actual RenzoOracle implementation and changed this back.